frontend/src/components/nodes/ValidatorRow.tsx

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import { Row, Col, OverlayTrigger, Tooltip } from "react-bootstrap";
+
+import { RpcConsumer } from "../utils/RpcProvider";
+import NodesApi, * as N from "../../libraries/explorer-wamp/nodes";
+
+import Timer from "../utils/Timer";
+import Balance from "../utils/Balance";
+
+interface Props {
+  node: any;
+}
+
+interface State {
+  nodeInfo?: N.NodeInfo;
+}
+
+export default class extends React.Component<Props, State> {
+  statusIdentifier = new Map([
+    ["AwaitingPeers", "Waiting for peers"],
+    ["HeaderSync", "Syncing headers"],
+    ["BlockSync", "Syncing blocks"],
+    ["StateSync", "Syncing state"],
+    ["StateSyncDone", "State sync is done"],
+    ["BodySync", "Syncing body"],
+    ["NoSync", ""],
+  ]);
+
+  state: State = {};
+
+  queryNodeInfo = async () => {
+    let nodeInfo = await new NodesApi().getValidatingInfo(
+      this.props.node.account_id
+    );
+    this.setState({ nodeInfo });
+  };
+
+  componentDidMount() {
+    this.queryNodeInfo();
+  }
+
+  render() {
+    const { node } = this.props;
+    const { nodeInfo } = this.state;
+    return (
+      <RpcConsumer>
+        {(context) => (
+          <Row className="node-row mx-0">
+            <Col md="auto" xs="1" className="pr-0">
+              {node.new ? (
+                <OverlayTrigger
+                  placement={"right"}
+                  overlay={
+                    <Tooltip id="new">
+                      next epoch upcoming validating nodes
```

Comment:
@kcole16 may I ask you to bikeshed the wording in the tooltips (search for `<Tooltip`) here? They are visible on the ["Validation Nodes" page](https://near-explorer-frontend-pr-349.onrender.com/nodes/validators) when you hover over the green (and other colors) dots on the left from the node ID:

![image](https://user-images.githubusercontent.com/304265/88162206-69bb7c00-cbe7-11ea-8ba1-810c41b4eeec.png)